examples/test_error.py

```diff
@@ -103,7 +101,7 @@
     # }}}
 
     # {{{Normalization
-    frq_slice = integrate_limits(select_pathway(s, signal_pathway), fl=fl)
+    frq_slice = integrate_limits(select_pathway(s, signal_pathway))
```

Comment:
these only show the signal energy plot, which is not informative.

Reply:
good call!